deploy/ui/form.json

```diff
@@ -1739,7 +1791,7 @@
                       "label": "Max Number of Session",
                       "type": "integer",
                       "required": false,
-                      "jsonPath": "$.spec.objects.servers[*].MDBMaxSession",
+                      "jsonPath": "$.spec.objects.servers[*].kieExecutorMDBMaxSession",
```

Comment:
please fix this in separate PR as for this bug there is opened another issue

Reply:
ok